app/windows/msi/build.js

```diff
@@ -0,0 +1,43 @@
+// You need to install wix tool set first
+// Reference https://ourcodeworld.com/articles/read/927/how-to-create-a-msi-installer-in-windows-for-an-electron-framework-application
```

Comment:
Can you please add the docs link?
https://github.com/felixrieseberg/electron-wix-msi#configuration